internal/connector/internal/services/connector_types.go

```diff
@@ -111,9 +154,7 @@ func (cts *connectorTypesService) Get(id string) (*dbapi.ConnectorType, *errors.
 	dbConn := cts.connectionFactory.New()
 
 	if err := dbConn.Unscoped().
-		Preload("Channels").
```

Comment:
It's implied in `Preload(clause.Associations)`, so we don't have to remember to preload associations individually. The type tests would have failed otherwise.